pkg/tracker/index.go

```diff
@@ -47,10 +47,9 @@ func GetIndexes() map[string][]*IndexTracker {
 func parseIndexFile(logger log.Logger, cfg *config.Config, DB db.DataServerProxy, client contracts.ETHClient) (trackersPerURL map[string]*IndexTracker, symbolsForAPI map[string][]string, err error) {
 
 	// Load index file.
-	indexFilePath := filepath.Join(cfg.ConfigFolder, "indexes.json")
-	byteValue, err := ioutil.ReadFile(indexFilePath)
+	byteValue, err := ioutil.ReadFile(filepath.Join("..", "..", cfg.ApiFile))
```

Comment:
It needs to parse the path literally
```suggestion
	byteValue, err := ioutil.ReadFile(cfg.ApiFile)
```